src/components/Home/ReusableKeyFeatures.jsx

```diff
@@ -0,0 +1,43 @@
+// React Imports
+import React from 'react';
+// Material UI Imports
+import { Box } from '@mui/system';
+import { Typography } from '@mui/material';
+
+const ReusableKeyFeatures = ({ icon, title, description }) => {
+  <>
+    <Box
```

Comment:
See my comment in the `Home.jsx` file of setting color for `Box` here, so we don't have to set it for each icon and this `Typogrpahy`.